lib/opts/index.ts

```diff
@@ -0,0 +1,122 @@
+import { getConfiguration as getConfigFile } from "../configuration";
+import { isin } from "../../type-helpers";
+import type { PrettyPrint } from "../../type-helpers";
+
+type Release = "minor" | "major" | "patch";
+type Task = "changelog" | "commit" | "tag";
+type Hook =
+  | "prerelease"
+  | "prebump"
+  | "postbump"
+  | "prechangelog"
+  | "postchangelog"
+  | "precommit"
+  | "postcommit"
+  | "pretag"
+  | "posttag";
+
+type TypePrefixes = Array<
+  (
+    | { section: string; hidden?: boolean | undefined }
+    | { hidden: true; section?: string | undefined }
+  ) & { type: string }
+>;
+
+type ConfigFiles = Array<
+  | string
+  | { filename: string; type: "json" | "gradle" | "plain-text" }
+  | { filename: string; updater: string }
+>;
+
+/**
+ * __THIS SHOULD NOT CHANGE.__
+ * @deprecated
+ *
+ * The configuration options for `standard-version` as of version 9.5 (The last version prior to the fork; deprecated).
+ */
+export type SVConfig = {
```

Comment:
This is purely personal preference, but I would prefer naming this `StandardVersionConfig`, as abbreviations and acronyms require you to remember whether you abbreviated or used an acronym (and which one).

Reply:
Sure thing! Personally, I'm indifferent on it - my rule on abbreviations is 'if you have a jsdoc tooltip explaining the type, no worries.' It's a personal preference, though, and it does require you to scroll the tool tip 🤷

Reply:
On second thought, how about LegacyConfig?

Reply:
Also good 👍